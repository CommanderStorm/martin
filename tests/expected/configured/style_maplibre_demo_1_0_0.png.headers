content-length: 97586
content-type: image/png
vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
