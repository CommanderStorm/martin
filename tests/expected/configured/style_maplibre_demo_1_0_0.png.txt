tests/output/configured/style_maplibre_demo_1_0_0.png: PNG image data, 512 x 512, 8-bit/color RGBA, non-interlaced
